inject/src/main/java/io/micronaut/context/DefaultBeanContext.java

```diff
@@ -3509,7 +3509,7 @@ private <T> void addCandidateToList(@Nullable BeanResolutionContext resolutionCo
             }
         } catch (DisabledBeanException e) {
             if (AbstractBeanContextConditional.ConditionLog.LOG.isDebugEnabled()) {
-                AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", beanType.getTypeName(), e.getMessage());
+                AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", beanType.getTypeName(), e.getMessage(), e);
```

Comment:
Stacktrace unlikely to add value here.
```suggestion
                AbstractBeanContextConditional.ConditionLog.LOG.debug("Bean of type [{}] disabled for reason: {}", beanType.getTypeName(), e.getMessage(), e);
```